help/destinations/catalog/advertising/pubmatic.md

```diff
@@ -123,6 +132,12 @@ Selecting target fields:
 
 ![Map attributes and identities](../..//assets/catalog/advertising/pubmatic/export-identities-to-destination.png)
 
+### Audience scheduling
+
+If you are using the PubMatic Connect (User Input for Segment Mapping) destination, you have to provide a mapping ID for each segment that corresponds to the 'Source Audience ID' in the PubMatic platform.
```

Comment:
excellent section and very clear, the only thing is if we could say audience instead of segment, please. 👍

Reply:
Also changed!